contrib/buildrpm.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+#set -vx
+
+PROGRAM="libs2n"
+VERSION="20150703"
+OPENSSLVER="1.0.2"
+
+BUILDNAME="${PROGRAM}-${VERSION}"
+SPECFILE="${PROGRAM}.spec"
+
+DIRECTORY=""
+RPMBUILD_ARGS="-ba"
+
+while [ $# -gt 0 ]; do
+	if [ -d "$1" ]; then
+		DIRECTORY=$1
+	fi
+
+	shift
+done
+
+if [ ! -d "$DIRECTORY" ]; then
+	printf "./%s <package directory>\n" `basename $0`
+	printf "\nExample: trunk/contrib/%s trunk\n" ${PROGRAM} `basename $0` ${PROGRAM}
+
+	exit 1
+fi
+
+if [ ! -f "$DIRECTORY/contrib/${SPECFILE}" ]; then
+	printf "Failed to locate SPEC file at %s.\n" "$1/contrib/${SPECFILE}"
+
+	exit 1
+fi
+
+DIRNAME="`dirname $DIRECTORY`"
+BASENAME="`basename $DIRECTORY`"
+SPECFULL="${BASENAME}/contrib/${SPECFILE}"
+CRYPTODIR="${BASENAME}/libcrypto-build"
+
+# Build local copy of libcrypto.
+cd ${CRYPTODIR} || (echo "Failed to change to libcrypto-build directory"; exit 1)
+echo -e "\nDownloading and building local copy of latest OpenSSL ${OPENSSLVER}...\n"
+curl -q https://www.openssl.org/source/openssl-${OPENSSLVER}-latest.tar.gz > openssl-${OPENSSLVER}.tar.gz
+tar -xzvf openssl-${OPENSSLVER}.tar.gz
```

Comment:
Considering packages are usually for non-experts, should we GPG verify the download?